xla/service/gpu/matmul_utils.h

```diff
@@ -151,7 +152,7 @@ struct GemmConfig {
         op.getBias() == nullptr ? nullptr : &bias_shape, GetShape(op.getD()),
         op.getAlphaReal().convertToDouble(),
         op.getAlphaImag().convertToDouble(), op.getBeta().convertToDouble(),
-        op.getAlgorithm(), compute_precision);
+        op.getAlgorithm(), compute_precision, false, false);
```

Comment:
nit: The code is easier to read if you annotate such parameters: `/*grad_x=*/false, /*grad_y=*/false`

Reply:
Done